docs/blog/2019-08-07-theme-jam/index.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Gatsby Theme Jam: Congratulations Vojtěch Holík and Allan Pooley!"
+date: 2019-08-07T12:00:00
+author: Jason Lengstorf
+excerpt: >
+  During the Theme Jam, 111 themes were created by our incredible community. Join us in celebrating the Gatsby community (that’s you!) and congratulating Vojtěch and Allan!
+tags:
+  - themes
+  - community
+---
+
+At the beginning of July, [Brent Jackson](https://twitter.com/jxnblk) joined _Learn With Jason_ to show [how to use `theme-ui`](https://youtu.be/6Z4p-qjnKCQ?list=PLz8Iz-Fnk_eTpvd49Sa77NiF8Uqq5Iykx). During the show, we announced the stable launch of [Gatsby themes](/docs/themes/) and kicked off the [Theme Jam](https://themejam.gatsbyjs.org), a celebration of our incredible community of creators.
+
+## What is the Theme Jam?
+
+![Theme Jam illustration by Maggie Appleton.](images/theme-jam.png)
+
+In a nutshell, the Theme Jam was a way for Gatsby to give back to its community: everyone who published a theme and submitted it to the Theme Jam will receive exclusive swag designed by [Maggie Appleton](https://maggieappleton.com/). Two creators who published themes our team reviewed as exemplary are getting an all-expenses paid trip to [Gatsby Days](https://www.gatsbyjs.com/resources/gatsby-days/). (For more details, see the [rules](https://themejam.gatsbyjs.org/rules).)
+
+## How were entries reviewed?
+
+Every Theme Jam entry needed to have a demo site, a published npm package, a public repo, and a properly configured `package.json`.
+
+Entries also needed to be incredibly performant (something something blazing fast) and meet accessibility standards.
+
+On top of that, submissions were code reviewed for documentation, the ability to extend and shadow the themes, and creativity.
+
+### Part 1: automated review
```

Comment:
```suggestion
### Part 1: Automated Review
```

Is there a reason these are lower-cased?

Reply:
Force of habit. I didn't check the style guide. My own style is to title case H2s, but sentence case H3 and below.